test/integration/eslint/test/next-lint.test.js

```diff
@@ -556,7 +556,9 @@ describe('Next Lint', () => {
     }
   })
 
-  test('format flag "compact" creates a file respecting the chosen format', async () => {
+  // TODO: unskip or remove it.
```

Comment:
These should just be moved to `test/development` and use our isolated test setup so that they can run with v8 and v9